server/src/main/java/org/elasticsearch/index/mapper/DateFieldMapper.java

```diff
@@ -382,7 +382,7 @@ public DateFieldMapper build(MapperBuilderContext context) {
             if (ft.name().equals(DataStreamTimestampFieldMapper.DEFAULT_PATH)
                 && context.isDataStream()
                 && ignoreMalformed.isConfigured() == false) {
-                ignoreMalformed.setValue(false);
+                ignoreMalformed.setValue(ignoreMalformed.getDefaultValue());
```

Comment:
I don't think this is correct? `@timestamp` can not ever have `ignore_malformed` because it is required in data streams.

Reply:
I pushed this just to see which test were failing...I am not going to merge this.